Resources/skeleton/serializer/document.mustache

```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <serializer>
-    <!--
-        This file has been generated by the EasyExtends bundle ( https://sonata-project.org/bundles/easy-extends )
-
-        @author <yourname> <youremail>
-    -->
+    <!-- This file has been generated by the SonataEasyExtendsBundle: https://sonata-project.org/bundles/easy-extends -->
 
```

Comment:
Should we remove all newlines too, because `xmllint` would remove them when calling `make lint`.